lib/pbench/agent/base.py

```diff
@@ -112,9 +112,7 @@ def verify_tool_group(self, group):
         try:
             self.tool_group_dir = self.gen_tools_group_dir(group)
         except BadToolGroup as exc:
-            click.echo(f'\t{self.name}: invalid --group option ("{group}"), {exc}')
-            ctxt = click.get_current_context()
-            click.echo(ctxt.get_help())
+            click.echo(f'{self.name}: invalid --group option ("{group}") -- {exc}')
```

Comment:
I'm not sure I like this `-- {exc}` style, if only because it clashes with the earlier `--group option` phrase. I don't particularly love the `,` either, but `:` might be better here.

Reply:
When I saw it printed out, the comma looked funny.  I was reaching for the colon, but then I noticed that there was already one in the line, and I feel that they don't nest.  So, I ended up with an "em dash".

I could change it to a single hyphen (or I could pull out the code point for a real unicode em dash 😉) or I could be convinced to go with a colon.

Reply:
We could remove the parens around the group name (it's already in quotation marks) and put them around the exception message...would that be good?

Reply:
Yeah, I like that a better ... parenthesizing the group name here seems weird anyway.

Reply:
Now that I've seen the message that the new formatting produces, I kinda like it.  🙂